samples/benchmarks/benchmarks.rst

```diff
@@ -0,0 +1,10 @@
+.. _benchmarks-sample:
```

Comment:
Here you are introducing `samples/benchmarks` directory -- this can be potentially misleading because we already have `tests/benchmarks` where we have various benchmark tests, and benchmarks are often considered "tests."

My suggestion would be to place the `coremark` benchmark application directory under `tests/benchmarks` or `samples/modules` (since this technically a "sample" exercising the coremark module).

Reply:
@stephanosio, I assume, that in future other benchmarks could be added to zephyr.
It to check what [eembc has](https://www.eembc.org/products/) - benchmarks that test different use cases for targets in the IoT topic. Sooner or later some more similar tests could appear here. Better to start group it from the beginning.

And the main purpose of this benchmark to evaluate and test target MCU and system performance.
Is the purpose of ```tests/benchmarks``` is the same? To not confuse.

Reply:
> Is the purpose of `tests/benchmarks` is the same?

More or less, yes.